applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
```

Comment:
Nit: Very thin line between this being a new file and a changed version of the previous one. Might update the year if this is considered a new file

Reply:
Most of the content is old so opt to keep as is.